packages/compass-web/src/connection-storage.spec.ts

```diff
@@ -20,8 +21,10 @@ const deployment = {
   ],
 };
 
+type Test = [string, ClusterDescriptionWithDataProcessingRegion, string];
```

Comment:
I couldn't get TypeScript to be happy with defining this `as const`, so I ended up just explicitly giving it a type.